__test__/api/organizations/createSingleOrganization.test.js

```diff
@@ -0,0 +1,67 @@
+const app = require("../../../server");
+const supertest = require("supertest");
+const OrganizationModel = require("../../../models/organizations");
+const request = supertest(app);
+
+describe("POST /organization", () => {
+  let organization;
+
+  afterEach(async () => {
+    await OrganizationModel.findByIdAndDelete(organization.organizationId);
+    organization = null;
+  });
+
+  test("should create an organization", async () => {
+    organization = {
+      organizationName: "Tesla Inc",
+      organizationEmail: "dev@tesla.com",
+      secret: "we_love_elon_haha",
+      adminName: "elon musk",
+      adminEmail: "iamelon@tesla.com",
+      adminPassword: "teslaspaceX123",
+    };
+    const res = await request.post("/v1/organizations").send(organization);
+
+    expect(res.status).toBe(201);
+    expect(res.body.status).toEqual("success");
+    expect(res.body.data).toContain(
+      '"organizationName" length must be at least 6 characters long in body'
```

Comment:
response should be similar to 
![image](https://user-images.githubusercontent.com/17856665/86942821-88c50300-c13d-11ea-9e79-b50016389b8b.png)